docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,284 @@
+        publishToNetwork:
+          description: |
+            If set, the node publishes this credential to the network. This is the default behaviour.
+            When set to false, the caller is responsible for distributing the VC to a holder. When the issuer is
+            also the holder, it then can be used to directly create a presentation (self issued).
+          type: boolean
+          default: true
+        visibility:
+            description: |
+              When publishToNetwork is true, the credential can be published publicly of privately to the holder.
+              This field is mandatory if publishToNetwork is true to prevent accidents.
+            type: string
+            enum: [ public, private ]
+            default: private
+        credentialSubject:
+          $ref: '#/components/schemas/CredentialSubject'
+    VerifiableCredential:
+      type: object
+      description: A credential according to the W3C and Nuts specs.
+      required:
+        - "@context"
+        - type
+        - issuer
+        - issuanceDate
+        - credentialSubject
+        - proof
+      properties:
+        "@context":
+          description: "List of URIs of JSON-LD contexts of the VC."
+          type: array
+          items:
+            type: string
+        id:
+          description: Credential ID. An URI wich uniquely identifies the credential e.g. the issuers DID concatenated with an uuid.
+          example: "did:nuts:123#B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY"
+          type: string
+        type:
+          description: List of type definitions for the credential.
+          type: array
+          items:
+            type: string
+        issuer:
+          $ref: '#/components/schemas/DID'
+        issuanceDate:
+          description: rfc3339 time string when the credential was issued.
+          type: string
+          example: "2012-01-02T12:00:00Z"
+        expirationDate:
+          description: rfc3339 time string untill when the credential is valid.
+          type: string
+          example: "2012-01-02T12:00:00Z"
+        credentialSubject:
+          $ref: '#/components/schemas/CredentialSubject'
+        proof:
+          description: one or multiple cryptographic proofs
+          type: object
+    ResolutionResult:
+      type: object
+      description: result of a Resolve operation.
+      required:
+        - verifiableCredential
+      properties:
+        revocationDate:
+          type: string
+          example: "2012-01-02T12:00:00Z"
+          description: If the credential is revoked, the field contains the revocation date.
+        verifiableCredential:
+          $ref: "#/components/schemas/VerifiableCredential"
+    Revocation:
+      type: object
+      description: Credential revocation record
+      required:
+        - issuer
+        - subject
+        - date
+      properties:
+        issuer:
+          $ref: '#/components/schemas/DID'
+        subject:
+          type: string
```

Comment:
subject might also be an object instead of a string?

Oh nevermind. I see this is the VC ID? A bit confusing.